external/substrate/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "substrate-minimal-runtime"
+version = "0.1.0"
+description = "Minimal Substrate runtime for testing compatibility"
+publish = false
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+frame-support = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/polkadot-sdk/", branch = "release-polkadot-v1.1.0" }
+sp-runtime = { version = "24.0.0", default-features = false, git = "https://github.com/paritytech/polkadot-sdk/", branch = "release-polkadot-v1.1.0", features = ["serde"] }
```

Comment:
```suggestion
frame-support = { version = "4.0.0-dev", default-features = false }
sp-runtime = { version = "24.0.0", default-features = false, features = ["serde"] }
```

Reply:
Before the recent shift to the monrepo, Substrate's been using polkadot release branches for versioning for years. I need to confirm whether, with the move to the monorepo we can version according to what's being suggested here.

Reply:
It's OK if we go with only git, but I don't think we need *both `git` and `version`.